packages/table/src/Cell/Cell.styles.ts

```diff
@@ -79,25 +79,33 @@ export const cellContentContainerStyles = css`
   display: flex;
   align-items: center;
   text-overflow: ellipsis;
-  transition: ${transitionDuration.default}ms ease-in-out;
-  transition-property: min-height, max-height, opacity, transform;
   min-height: ${standardCellHeight}px;
+  transition-property: min-height, max-height, opacity, transform, padding;
+  transition-duration: ${transitionDuration.slowest}ms;
+`;
+
+export const disableAnimationStyles = css`
+  transition: none;
 `;
 
 const _hiddenStyles = css`
   opacity: 0;
   min-height: 0;
   max-height: 0;
+  padding: 0;
 `;
 
 export const cellContentTransitionStyles: Record<TransitionStatus, string> = {
   entered: css`
     opacity: 1;
     min-height: ${standardCellHeight}px;
-    max-height: ${standardCellHeight}px;
+    max-height: 30vh;
```

Comment:
This is still causing a choppy animation.
1. Can you add a story that reproduces the problem this is solving?
2. If we want the cells to animate to the size of the content, we should probably follow the pattern we use in `ExpandableContent` to make sure the height is correct: https://github.com/mongodb/leafygreen-ui/blob/LG-3439-cell-max-height-2/packages/table/src/ExpandedContent/ExpandedContent.tsx#L40-L45